### PR TITLE
- Refactoring

### PR DESCRIPTION
**Pull request by gkaretka**
Signed-off-by: Gregor Karetka gregorkaretka999@gmail.com

### CODE REVIEW

File: example/example.ino~ [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include <sorter.h>
```
Comment (pepol) [inline example/example.ino~]: Please delete this file. Backups (files ending with `~`) should not be part of git repository.

File: example/example.ino [hunk 2]
```diff
@@ -13,7 +13,10 @@ void loop() {
             a[i][1] = i;
         }
   
-        sorter::sort(a);
+        //sorter::sort_max(a); // Set two first indexes in array to max values
+        sorter::sort_min(a); // Set two first indexes in array to min values
```
Comment (pepol) [inline example/example.ino]: Please always separate code and comments with 2 spaces.

**Comment by pepol** [comment]
@magic-sudo looks good, please address my comments.

File: example/example.ino~ (hunk 1 above)
Comment (gkaretka) [inline example/example.ino~]: done

File: example/example.ino (hunk 2 above)
Comment (gkaretka) [inline example/example.ino]: done

**Comment by pepol** [comment]
LGTM, please squash this. Put: `sorter: Add function to find minimums.`

**Pull request merged**